modules/endorspy/main.nf

```diff
@@ -0,0 +1,39 @@
+process ENDORSPY {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::endorspy=0.4" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/endorspy:0.4--hdfd78af_0':
+        'quay.io/biocontainers/endorspy:0.4--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(stats)
+    tuple val(meta), path(stats_optional)
```

Comment:
Technically as the two flagstats come from the same sample, you can put this as the same input tuple channel (to make sure they are linked together), but you can get around this with a `.multiMap` (for reference)

Reply:
@aidaanva, do you have a response to this? @jfy133 , do you want to require this?

Reply:
We are looking together at this tomorrow or Thursday!